.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "gitsubmodule"
     directory: "/"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
       day: "friday"
       time: "06:00"
     assignees:
```

Comment:
You could remove the assignees everywhere in this file. 
I have notification turned on anyway for all of our repositories and that way it's one email less per PR.

Reply:
:+1: